apps/cli-daemon/src/app/workspace-manager/dto/directory.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { IsNotEmpty, IsOptional } from 'class-validator';
+
+export class DirectoryDto {
+  @IsNotEmpty()
+  name: string;
+
+  @IsNotEmpty()
+  isNG: boolean;
+
+  @IsOptional()
+  isFavorite: boolean;
+
+  constructor(name: string, isNG = false, isFavorite = false) {
```

Comment:
have I missed this file?
you can use the "private" modifier to set those instead doing
this.x = x

Reply:
should have been in another PR my bad.

Reply:
I don't think it's possible to mix modifiers and class validators...
if I pass the validator in the constructor I get an error

Reply:
@Tcodrz oh you right